skactiveml/visualization/multi/_multi_annot_feature_space.py

```diff
@@ -8,7 +8,7 @@
 from skactiveml.utils import is_labeled, check_scalar
 from .._feature_space import plot_decision_boundary
 from ...utils._validation import check_type
-from ...utils._visualisation import mesh, check_bound, _get_contour_args, \
+from skactiveml.visualization._auxiliary_functions import mesh, check_bound, _get_contour_args, \
```

Comment:
relative import should be used, i.e., ..visualization._auxiliary_functions import ...